### ISSUE TITLE
Typescript support

### ISSUE THREAD

**Issue by pr0ton** [OpenCageData/geosearch#268]
No typescript support exists, either via the Definitely Typed `@types` package or through using typescript itself.

```
Could not find a declaration file for module '@opencage/geosearch-core'. /<redacted>/node_modules/@opencage/geosearch-core/dist/js/opencage-geosearch-core.cjs.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/opencage__geosearch-core` if it exists or add a new declaration (.d.ts) file containing `declare module '@opencage/geosearch-core';`
```

**Comment by tsamaya** [comment]
Hi @pr0ton, please bear with me I will add the definition file for the core package

**Comment by tsamaya** [comment]
The new release `0.1.0` (https://www.npmjs.com/package/@opencage/geosearch-core) contains a built-in typescript declaration.

You can also check the declaration file here: https://github.com/OpenCageData/geosearch/blob/master/packages/geosearch-core/index.d.ts.

please let me know if there is anything I can help with.

**Comment by tsamaya** [comment]
@pr0ton I updated the DTS, as I forgot the only mandatory parameter `key`. This is now fixed on 0.1.1. 

I keep the issue open for a few days. Please let me know if there is anything we can help with.

**Comment by pr0ton** [comment]
Thanks. This works well.
